src/components/Tabs.react.js

```diff
@@ -229,6 +320,69 @@ export default class Tabs extends Component {
             });
         }
 
+        if (this.props.children) {
+            nextTab = () => {
```

Comment:
This doesn't need to be a function, just create the instance of `NextTab` here directly